subsys/net/lib/lwm2m/Kconfig

```diff
@@ -47,7 +47,7 @@ config LWM2M_COAP_BLOCK_SIZE
 	range 64 1024
```

Comment:
Regarding this commit, plase note that every commit requires some body in Zephyr/NCS, even if it's trivial.

Reply:
Time to check all commits separately with gitlint.

```
...:~/ws/ncs2/zephyr$ gitlint 
1: UC3 Title does not follow [subsystem]: [subject] (and should not start with literal subsys:): "subsys: net: lib: lwm2m: Decouple CoAP message..."
```

Reply:
Right, I didn't notice, `subsys:` in the prefix is not allowed :)